.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: python
 python: 2.7
 install:
-  - pip install -r requirements.txt
+  - pip install flake8
```

Comment:
I think we can change this to use pipenv. Perhaps `pip install pipenv` for the install and `pipenv check` and `pipenv check --style .` for the script?